types/paradisebotsapi.js/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for paradisebotsapi.js 1.0
+// Project: https://github.com/ParadiseBotList/paradisebotsapi.js#readme
+// Definitions by: Yoshida Tomio <https://github.com/1chiSensei>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class PBL {
+    constructor(id?: string, auth?: string);
```

Comment:
```suggestion
    constructor(id: string, auth: string);
```
https://github.com/ParadiseBotList/paradisebotsapi.js#arguments
those cannot be optional IMO, one is used in string template, other in authorization header.

Reply:
can you test this using runtime code? I can see README using parameterless constructor.
The git history shows those were required:
https://github.com/ParadiseBotList/paradisebotsapi.js/commit/6b64a5dd171368786cb010c09d73ea5a925daec8#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R5-R6

Reply:
Sensible question.

Reply:
but you wont need this when running `.get()` and `.getUser()`

Reply:
no worries, it works, tested locally:
```js
new PBL()
PBL { id: undefined, auth: undefined }
```